src/test/java/org/folio/rest/api/LoanPoliciesApiTest.java

```diff
@@ -76,6 +78,7 @@ public void loanPolicyWithFixedDate() throws
 
     //////////// create loan policy with foreign key to fdd
     JsonObject loanPolicyRequest = new LoanPolicyRequestBuilder()
+      .defaultPolicy()
```

Comment:
Why is the default policy not expressed in the default constructor of the builder, rather than a separate method?

Reply:
the defaultPolicy() method constructs a rolling policy with default rolling-specific fields (periods) set. 

And in case we need a fixed policy, we will have to reset those rolling specific fields first, otherwise we will get a validation error.

Reply:
I think I'm confused. My question was not about what the definition of the default policy is, what I'm curious about is why make clients of the `LoanPolicyRequestBuilder` have to explicitly ask for that, rather than just define it in the constructor?

What happens if the `defaultPolicy` method is not called on the builder, what policy is defined?

Reply:
So, we have two types of policies - fixed and rolling. Each type has different set of properties (periods vs FDDS).

If we construct rolling policy in the default builder's constructor, then in order to build a fixed policy we will have to 1st reset all rolling specific fields, and 2nd set the fields that we're interesting in.

Because of this, the default constructor sets values to the common fields only - name, description and ID but Fixed and Rolling specific fields keeps unassigned.

Reply:
> So, we have two types of policies - fixed and rolling. Each type has different set of properties (periods vs FDDS).

That's a fair point. In which case I suggest we change the name of the method to `defaultRollingPolicy` or something like that.

> If we construct rolling policy in the default builder's constructor, then in order to build a fixed policy we will have to 1st reset all rolling specific fields, and 2nd set the fields that we're interesting in.

If there is no single sensible default, then I suggest making that constructor private (or getting rid of it) and converting the `defaultPolicy` to a static factory method (and renamed as above).

> Because of this, the default constructor sets values to the common fields only - name, description and ID but Fixed and Rolling specific fields keeps unassigned.

That suggests there is no need for the default constructor

Reply:
done